libqtile/scripts/migrate.py

```diff
@@ -125,6 +125,59 @@ def windowtogroup_groupName_argument(funcname, query):  # noqa: N802
     return query.select_method(funcname).modify_argument("groupName", "group_name")
 
 
+def command_decorators_changes(query):
+    """
+    Some commands were renamed when moving from `cmd_` to decorator syntax for
+    exposed commands.
+
+    While most code should continue to work, with required changes indicated in log files,
+    some changes may cause breakages.
+
+    This migration function attempts to address the key changes.
+    """
+    return (
+        query.select_method("cmd_groups")  # noqa: BLK100
+        .rename("get_groups")
+        .select_method("cmd_screens")
+        .rename("get_screens")
+        .select_method("opacity")
+        .rename("set_opacity")
+        .select_method("cmd_opacity")
+        .rename("set_opacity")
+        .select_method("hints")
+        .rename("get_hints")
+        .select_method("cmd_hints")
+        .rename("get_hints")
+    )
```

Comment:
Could we also add these? They weren't commands but were also updated.
```suggestion
        .select_method("getsize")
        .rename("get_size")
        .select_method("getposition")
        .rename("get_position")
    )
```

Reply:
We can but I don't like these migrate scripts. What if someone has defined their own `getsize` function?

Reply:
The `cmd_` ones will still work. Users will just get a warning telling them to update their code.

Reply:
OK, nvm my suggested ones